Cargo.toml

```diff
@@ -82,11 +82,11 @@ correctness = "deny"
 dbg_macro = "deny"
 
 [profile.fast-test]
-inherits = "dev"
+inherits = "test"
 debug = 1
 
 [profile.fast-test.package."*"]
-opt-level = 3
```

Comment:
I don't think it's job of the tests to work out whether a release build works. We separately build Holochain in release mode for the Holonix tests which should check that it builds and starts.

All I wanted to do here was speed up how long it takes to link and optimise the tests. It was spending a lot of time optimising things on my system and I couldn't tell a significant different in execution speed because the test binaries are full of other debug information that makes them slower than a real release binary anyway.

What maackle changed on the other PR is much more extreme and I suspect it's causing OOMs because it's fully release optimising every release binary. Just doing that for Holochain without all the test utilities and multiplier of doing it for many tests, takes about 4 minutes and a lot of memory. We can probably tune things a bit for performance on CI but it looks like that PR went too far to me.

Reply:
thanks for clarifying. i wasn't clear whether this was required to workaround a test assertion that didn't succeed otherwise. i understand now that it's related to build-time resource usage and that's doesn't seem concerning to me; so your clarification unworried me!